atkinson/config/search.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python
+"""Search tools for configuration files"""
+
+from __future__ import print_function
+
+import os
+
+
+def _check_available(filename):   # pragma: no cover
+    """Check to see if the filename exists and is a file"""
+    return os.path.exists(filename) and os.path.isfile(filename)
+
+
+def expand_user(path):
+    """Expand ~ to /home/<username>"""
+    if '~' in path:
+        path = str.replace(path, '~', os.path.expanduser('~'))
```

Comment:
I thought path.expanduser could take any string and do what this function is trying to do.  Is there a case that doesn't work, or is this doing something different than it sounds like?

Reply:
It appears that it does now, I will remove that function and update the code to use os.path.expanduser.